sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -7,8 +7,10 @@
 - Added the `Languages` property to the `AnalyzeResult` class. This property is populated when using the `prebuilt-read` model and holds information about the languages in which the document is written.
 - Added the `Tags` property to the `BuildModelOptions` class. This property can be used to specify custom key-value attributes associated with the model to be built.
 - Added the `Tags` property to the `DocumentModelInfo` and to the `ModelOperationInfo` classes.
+- Added the `BuildMode` property to `DocTypeInfo` to indicate the technique used when building the correspoding model.
 
 ### Breaking Changes
+- Added the required parameter `buildMode` to `StartBuildModel` methods. Users must now choose the technique (`Template` or `Neural`) used to build models.
```

Comment:
As far as I'm aware, we don't have service documentation for the build modes yet. Ideally I would link it here too. Created issue https://github.com/Azure/azure-sdk-for-net/issues/26780 so we don't forget to add the link everywhere when we have it.

Reply:
Do we have an aka.ms link for this yet? can't remember if someone created one already

Reply:
I don't think so. Sameeksha has just asked about it in the v-team chat, so we may have one soon.

Reply:
Copying it from her PR 😆 

https://aka.ms/azsdk/formrecognizer/buildmode

Thanks @samvaity!